agent/consul/client_test.go

```diff
@@ -490,6 +492,14 @@ func newClient(t *testing.T, config *Config) *Client {
 	return client
 }
 
+func newTestResolverConfig(t *testing.T, suffix string) resolver.Config {
+	n := t.Name()
+	s := strings.Replace(n, "/", "", -1)
+	s = strings.Replace(s, "_", "", -1)
+	return resolver.Config{Authority: strings.ToLower(s) + "-" + suffix}
+}
+
+// TODO(rb): add tests for the wanfed/alpn variations
```

Comment:
Did you want to do that in this PR?

Reply:
Actually in this specific spot it doesn't make sense to test RPC via ALPN because as it all exists today that's only executed for Server-to-Server and this file is testing Client-to-Server which currently does not do ALPN.